features/networking/sgw-lgw-migration.feature

```diff
@@ -0,0 +1,113 @@
+Feature: SGW<->LGW migration related scenarios
+  
+  # @author anusaxen@redhat.com
+  # @case_id OCP-47561
+  @4.10
+  @admin
+  @destructive
+  @network-ovnkubernetes
+  @vsphere-ipi
+    Scenario: [SDN-2290] SGW <-> LGW migration scenario	for vsphere platform
+    ######## Prepare Data Pre Migration for multiple use cases############
+    Given the env is using "OVNKubernetes" networkType
+    
+    #OCP-47087 - [bug_1903408]Other node cannot be accessed for nodePort when externalTrafficPolicy is Local	
+    Given I store the masters in the :masters clipboard
+    And the Internal IP of node "<%= cb.masters[0].name %>" is stored in the :master0_ip clipboard
+    And evaluation of `rand(30000..32767)` is stored in the :port clipboard
+    Given I have a project
+    Given I obtain test data file "networking/nodeport_test_pod.yaml"
+    When I run the :create client command with:
+      | f | nodeport_test_pod.yaml |
+    Then the step should succeed
+    Given a pod becomes ready with labels:
+      | name=hello-pod |
+    And evaluation of `pod.node_ip` is stored in the :hostip clipboard
+    When I obtain test data file "networking/nodeport_test_service.yaml"
+    When I run oc create over "nodeport_test_service.yaml" replacing paths:
+      | ["spec"]["ports"][0]["nodePort"]  | <%= cb.port %> |
+      | ["spec"]["externalTrafficPolicy"] | Local          |
+    Then the step should succeed
+    
+    #Switching cluster to another gateway mode and reverting back to original in clean up
```

Comment:
@anuragthehatter Current testing is to test SGW<->LGW migration, then revert the mode from LGW to SGW, should we run the curl testing step one more time to make sure feature still work when LGW to SGW?